app/helpers/user_journey_helper.rb

```diff
@@ -2,4 +2,16 @@ module UserJourneyHelper
   def display_component(certificate)
     certificate.component_type == COMPONENT_TYPE::MSA ? COMPONENT_TYPE::MSA_SHORT : COMPONENT_TYPE::VSP_SHORT
   end
+
+  def primary_signing_certificate(certificate)
+    certificate == certificate.component.signing_certificates.reverse.first
+  end
+
+  def secondary_signing_certificate(certificate)
+    certificate == certificate.component.signing_certificates.reverse.second
+  end
+
+  def position(certificate)
+    certificate == certificate.component.signing_certificates.reverse.first ? 'primary' : 'secondary'
```

Comment:
```suggestion
    certificate == primary_signing_certificate(certificate) ? PRIMARY : SECONDARY
```

Can we use constants?

Reply:
This is the only place we are directly using the words, so would we want to use constants for a single occurrence?